### DISCUSSION TITLE
http://localhost:1337/graphql/ don't work

### DISCUSSION THREAD

**Discussion by ricar2ce** [strapi/strapi#5871] [Legacy Topics]
I have running strapi, but when I go to the address: http://localhost:1337/graphql/ , it returns the following:
{"statusCode": 404, "error": "Not Found", "message": "Not Found"}

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by abanchev** [comment]
Did you install the graphql plugin from the admin panel.

**Reply by ricar2ce** [reply]
yes

**Comment by lauriejim** [comment]
Are you starting your application in production environment?
